### PR TITLE
Enhancements and Remove legacy support

### PR DESCRIPTION
**Pull request by cstoobes**
* Add additional island classes.
* Update button styles.
* Remove link underline styles.
* Remove support for IE10/11.
* Remove placeholder mixin with vendor prefixes.

### CODE REVIEW

**Comment by spacestierman** [comment]
👍 looks great to me!

**Review by bobkellyspace150 — APPROVED** [review]

**Review by spacestierman — APPROVED** [review]

File: scss/ui/_island.scss [hunk 1]
```diff
@@ -3,28 +3,42 @@
...
-  margin-bottom: 2 * $base-spacing-unit;
+  margin-bottom: $base-spacing-unit;
+}
+
+.landmark--dubs {
+  margin-bottom: $base-spacing-unit * 2;
 }
 
 .landmarklet {
   margin-bottom: $base-spacing-unit;
```
Comment (bobkellyspace150) [inline scss/ui/_island.scss:39]: If `.landmark` is now just `$base-spacing-unit`, should `.landmarklet` be `$half-spacing-unit`?

Comment (spacestierman) [reply scss/ui/_island.scss:39]: Good catch! IMO, the `*let` classes (`island`, `landmarklet`) should have the same spacing.

Comment (cstoobes) [reply scss/ui/_island.scss]: yes, great catch!

**Pull request merged**